### PR TITLE
Switched email to another user to use display API sendSetupEmail

### PR DESCRIPTION
**Pull request by alexey-rise**
@alex-deaconu let's try this again :-)

### CODE REVIEW

File: test/unit/displays/controllers/ctr-display-add-modal.tests.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ describe('controller: display add modal', function() {
       $scope.anotherEmail = 'another@email.com';
       $scope.sendToAnotherEmail();
 
-      spy.should.have.been.calledWith('ID','Name','another@email.com');
+      spy.should.have.been.calledWith('ID', 'another@email.com');
```
Comment (alex-deaconu) [inline test/unit/displays/controllers/ctr-display-add-modal.tests.js]: @alexey-rise wondering if you could clean up the `$scope.display.name` variable above, it's no longer needed. Thanks.

Comment (alexey-rise) [reply test/unit/displays/controllers/ctr-display-add-modal.tests.js]: @alex-deaconu done, see last commit.

File: web/scripts/displays/services/svc-display-email.js [hunk 2]
```diff
@@ -1,26 +1,20 @@
 'use strict';
 
 angular.module('risevision.displays.services')
-  .service('displayEmail', ['$templateCache', 'userState', 'email', '$q',
-    function ($templateCache, userState, email, $q) {
+  .service('displayEmail', ['$templateCache', 'userState', 'display', '$q',
+    function ($templateCache, userState, displayService, $q) {
```
Comment (alex-deaconu) [inline web/scripts/displays/services/svc-display-email.js]: @alexey-rise the service name and variable name do not match (`display` vs `displayService`) which could cause confusion.

Also, wondering if you could clean up `$templateCache` as it's no longer used/needed from here and the unit tests. 

Thanks.

Comment (alexey-rise) [reply web/scripts/displays/services/svc-display-email.js]: @alex-deaconu please see the last commit.

File: test/unit/displays/services/svc-display-email.tests.js [hunk 3]
```diff
@@ -3,9 +3,9 @@ describe('service: displayEmail:', function() {
...
 
   beforeEach(module(function ($provide) {
-    $provide.service('email', function() {
+    $provide.service('display', function() {
       return {
-        send: function() {
+    	sendSetupEmail : function(){
           return Q.resolve();
         }
       }
```
Comment (alex-deaconu) [inline test/unit/displays/services/svc-display-email.tests.js]: @alexey-rise $templateCache needs to be removed from here as well. Thanks.

File: web/scripts/displays/services/svc-display.js [hunk 4]
```diff
@@ -307,6 +307,27 @@
...
+                });
+              })
+              .then(function (resp) {
+                $log.debug('sendSetupEmail resp', resp);
+                deferred.resolve(resp);
+              })
+              .then(null, function (e) {
+                console.error('Failed to send setup email.', e);
+                deferred.reject(e);
+              });
```
Comment (alex-deaconu) [inline web/scripts/displays/services/svc-display.js]: @alexey-rise wondering if you could add a basic test for this function. Thanks.

Comment (alex-deaconu) [reply web/scripts/displays/services/svc-display.js]: @alexey-rise I meant a test of svc-display's sendSetupEmail.

**Comment by alexey-rise** [comment]
@alex-deaconu please review last 2 commits.

**Comment by alexey-rise** [comment]
@alex-deaconu fixed and squashed, please review.

**Comment by alex-deaconu** [comment]
@alexey-rise looks good, thanks for the fixes 👍

**Pull request merged**